pandas/core/groupby/groupby.py

```diff
@@ -1109,20 +1109,10 @@ def _cython_agg_general(
             result = self.grouper._cython_operation(
                 "aggregate", obj._values, how, axis=0, min_count=min_count
             )
-
-            if how == "ohlc":
-                # e.g. ohlc
-                agg_names = ["open", "high", "low", "close"]
-                assert len(agg_names) == result.shape[1]
-                for result_column, result_name in zip(result.T, agg_names):
-                    key = base.OutputKey(label=result_name, position=idx)
-                    output[key] = result_column
-                    idx += 1
-            else:
-                assert result.ndim == 1
-                key = base.OutputKey(label=name, position=idx)
-                output[key] = result
-                idx += 1
+            assert result.ndim == 1
+            key = base.OutputKey(label=name, position=idx)
+            output[key] = result
+            idx += 1
```

Comment:
There's an OHLC specific comment on L1098 that probably isn't relevant anymore (suggests using `enumerate`, but I guess that's still not possible because of the `continue` early in the loop).

Reply:
yah, i was counting that as part of the "simplify _wrap_aggregated_output" follow-up